src/user/user.mapper.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from "@nestjs/common";
+
+import { Task } from "src/task/task.entity";
+import { UserDTO } from "./user.dto";
+import { User } from "./user.entity";
+
+@Injectable()
+export class UserMapper {
+
+	public toDTO(entity: User): UserDTO {
+		return {
+			id: entity.id,
+			email: entity.email,
+			password: entity.password,
+			name: entity.name,
+			lastLoginAt: entity.lastLoginAt,
+			tasks: entity.tasks?.map(task => task.id)
+		};
+	}
+
+	public toDTOs(entities: User[]): UserDTO[] {
+		const userDTOs: UserDTO[] = [];
+		for (const entity of entities) {
+			userDTOs.push(this.toDTO(entity))
+		}
+		return userDTOs;
+	}
+
+	public toEntity(dto: UserDTO): User {
```

Comment:
you can delete this method if it is not used

Reply:
it's used in TaskMapper's toEntity method